x-pack/plugins/security_solution/public/management/pages/policy/view/policy_advanced.tsx

```diff
@@ -37,9 +47,17 @@ function getValue(obj: Record<string, unknown>, path: string[]) {
   return currentPolicyConfig[path[path.length - 1]];
 }
 
+const warningMessage = `This section contains policy values that support advanced use cases. If not configured
+properly, these values can cause unpredictable behavior. Please consult documentation
+carefully or contact support before editing these values.`;
+
 export const AdvancedPolicyForms = React.memo(() => {
   return (
     <>
+      <EuiCallOut title="Proceed with caution!" color="warning" iconType="help">
```

Comment:
@caitlinbetz @bfishel how do you feel about this title/style? see screenshot in PR body

Reply:
@jmiller263 

can you change the `iconType` to `alert` ?

![image](https://user-images.githubusercontent.com/56395104/101215848-7e575880-364c-11eb-8d4b-6aa41881344b.png)

EUI Icon types: https://elastic.github.io/eui/#/display/icons

Reply:
done 8d25f9803ab7120d63895767a37190729ce4173e